cognite_toolkit/cdf_tk/bootstrap.py

```diff
@@ -32,34 +33,34 @@
 
 @dataclass
 class AuthVariables:
-    cluster: str
-    project: str
-    token: str
-    client_id: str
-    client_secret: str
-    cdf_url: str = None
-    token_url: str = None
-    tenant_id: str = None
-    audience: str = None
-    scopes: str = None
+    cluster: str | None
+    project: str | None
+    token: str | None
+    client_id: str | None
+    client_secret: str | None
```

Comment:
I set these to `None` to reflect the behavior as it is today. The other option, is to raise an exception in line 54-58 if any of these are missing from the environment.

Reply:
Let's go for None for now. If we raise an exception, we might want to use the typer support for prompting if a command line option is not set either from env var or explicitly.